tests/test_tag.py

```diff
@@ -155,3 +155,44 @@ def test_push_tag_failed(mock_repo):
     with patch('git.repo.fun.name_to_object'):
         with pytest.raises(exceptions.PushException):
             repo.tag.push("my_tag", "origin")
+
+
+def test_list_tags(mock_repo):
+    """
+    GIVEN GitRepo is initialized with a path and repo
+    WHEN GitTag.list_tags() is called
+    THEN repo.tags is returned
+    """
+    class test_tag(object):
```

Comment:
In general, class names should use CapitalisedWords as per pep8 conventions https://www.python.org/dev/peps/pep-0008/#class-names